### PR TITLE
Release v1.8.0

### PR DESCRIPTION
**Pull request by carlosalberto**
Monthly release for November.

### CODE REVIEW

**Review by jsuereth — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -45,8 +69,12 @@ release.
 
 ### Resource
 
+- No changes.
+
 ### Semantic Conventions
```
Comment (arminru) [inline CHANGELOG.md:74]: https://github.com/open-telemetry/opentelemetry-specification/commit/a25d5f03ab58ecf88c09f635df97d2328b5ba237 could be added

**Review by arminru — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (carlosalberto) [reply CHANGELOG.md:80]: Done

[the remark this answers is not in the document]
Comment (carlosalberto) [reply CHANGELOG.md:80]: Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +39,8 @@ release.
 
 ### Traces
 
+- No changes.
+
 ### Metrics
```
[the remark this answers is not in the document]
Comment (carlosalberto) [reply CHANGELOG.md:44]: Added

**Review by MrAlias — APPROVED** [review]

**Pull request merged**